{"status": "broken", "name": "MATCH and OPTIONAL MATCH on same pattern", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MatchAcceptance2"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486064589, "historyId": "27dd434cbaaf8f27ff6ed04238646b0d", "start": 1578486064368, "steps": [{"status": "passed", "start": 1578486064368, "stop": 1578486064579, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486064579, "stop": 1578486064583, "name": "And having executed", "attachments": [{"source": "87c9c286-102d-4930-8770-d4499e98639f-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486064584, "stop": 1578486064586, "name": "When executing query", "attachments": [{"source": "b0e3c2af-3dcf-4f93-ac5f-eb5bdf02af2b-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "a96c6666-f7df-493b-94fd-737b02921f24-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486064588, "start": 1578486064586, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support has labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486064589, "stop": 1578486064589, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support has labels\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "2859e096-8eb0-4d42-8ee6-f7307f035b1c"}